aspnetcore/security/authentication/2fa.md

```diff
@@ -21,7 +21,7 @@ uid: security/authentication/2fa
 
```

Comment:
We have successfully tested it with VS 2017 RC4 / Core 1.0/1.1 and converted the Warning to a Note.